lzy/long-running/src/main/java/ai/lzy/longrunning/OperationRunnerBase.java

```diff
@@ -43,52 +47,61 @@ protected OperationRunnerBase(String id, String descr, Storage storage, Operatio
 
     @Override
     protected final void execute() {
-        try {
-            if (!loadOperation()) {
-                return;
-            }
+        LogUtils.withLoggingContext(prepareLogContext(), () -> {
```

Comment:
we have the same logic in the parent `ContextAwareTask`
let's merge it

Reply:
Merge - like how?

Reply:
we can prepare logging context in the ctor